rust/protocol/src/sealed_sender.rs

```diff
@@ -1194,6 +1569,12 @@ impl SealedSenderDecryptionResult {
     }
 }
 
+/// Decrypt a Sealed Sender message `ciphertext` in either the v1 or v2 format, and validate its
+/// sender certificate.
+///
+/// This method calls [sealed_sender_decrypt_to_usmc] to extract the sender information, including
+/// the embedded [SenderCertificate]. The sender certificate is then validated against the
+/// `trust_root` provided by the backend server to ensure that the sender's identity was not forged.
```

Comment:
I think it's worth adding something like "For more details on the sealed sender formats, see [`sealed_sender_encrypt_from_usmc`] and [`sealed_sender_multi_recipient_encrypt`]."

Reply:
`trust_root` isn't actually provided by the server; it's baked into the apps. The root signs the server certificate, which signs the sender certificate.

Reply:
Thank you for this very very helpful clarification!! I have added both of these comments here.